changelog/v0.36.5/add-target-ref.yaml

```diff
@@ -0,0 +1,4 @@
+changelog:
```

Comment:
nit: since this is breaking, you could move to v0.37.0 and use type BREAKING_CHANGE

Reply:
the goal was to fix this before it became a breaking change as no one has consumed this message yet (see PR description for references to edge & GME)
the versioning is a bit flimsy in this repo (e.g. there is currently not a `v0.36.x` brach) so it seemed easiest to just break it now while there are no consumers.

that being said, it is technically a break from a semver perspective, so I'm not sure the best approach